src/test/java/com/wikia/webdriver/testcases/discussions/PostsListSharing.java

```diff
@@ -0,0 +1,85 @@
+package com.wikia.webdriver.testcases.discussions;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.discussions.PostsListPage;
+
+import org.testng.annotations.Test;
+
+/**
+ * @ownership Content X-Wing Wikia
+ */
+
+@Test(groups = {"Discussions", "PostListSharing"})
+public class PostsListSharing extends NewTestTemplate {
+
+  private static final String DESKTOP_RESOLUTION = "1366x768";
+  private static final String MOBILE_RESOLUTION = "600x800";
+
+  private Credentials credentials = Configuration.getCredentials();
+
+  /**
+   * ANONYMOUS USER SECTION
+   */
+
+  @Test
+  @Execute(asUser = User.ANONYMOUS)
+  @InBrowser(browserSize = MOBILE_RESOLUTION)
+  public void socialNetworkIconsAreDisplayedToAnonymousUserOnMobile() {
+    toggleShareIconClickDisplaysSocialNetworkIcons(
+        new String[]{"Facebook", "Twitter", "Reddit", "Tumblr"});
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Twitter" 13 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Reddit" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Facebook" 14 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Tumblr" 9 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)